src/store/__mocks__/load-data.js

```diff
@@ -37,6 +38,11 @@ const loadJsonData = async (path = getUrl('main')) => {
     return mockAPIFeatureSupport(animals);
   }
 
+  // Use nodes_task dataset in place of 'main' endpoint
+  if (path.includes('nodes')) {
+    return mockAPIFeatureSupport(node_task);
+  }
+
```

Comment:
Great :)

It might be worth seeing if you can find a way to let it return any one of the three mock node data types, depending on the URL that's passed to it?

Reply:
Also, I don't think this one needs to be wrapped in `mockAPIFeatureSupport`, as that won't be used for this endpoint, it's just for pipeline data.